Index: node_modules/irc/lib/irc.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- node_modules/irc/lib/irc.js	(revision )
+++ node_modules/irc/lib/irc.js	(revision )
@@ -581,7 +581,7 @@
 
             case 'err_erroneusnickname':
                 if (self.opt.showErrors)
-                    util.log('\033[01;31mERROR: ' + util.inspect(message) + '\033[0m');
+                    util.log('ERROR: ' + util.inspect(message));
                 self.emit('error', message);
                 break;
 
@@ -1021,7 +1021,7 @@
 };
 
 Client.prototype.ctcp = function(to, type, text) {
-    return this[type === 'privmsg' ? 'say' : 'notice'](to, '\1' + text + '\1');
+    return this[type === 'privmsg' ? 'say' : 'notice'](to, text);
 };
 
 Client.prototype.convertEncoding = function(str) {
